### ISSUE TITLE
add a way to install the program

### ISSUE THREAD

**Issue by sheeepdev** [Oxygem/Kanmail#151]
hi! i discovered this mail client after hours of searching online for a good, cozy UI and open-source client :)

The only thing I'm missing is a way to install the program, its currently a portable .exe, I cant find an installer anywhere so I'm submitting this feature request

**Comment by Fizzadar** [comment]
Hi @sheeepdev glad to hear you like Kanmail! I like this idea - I'm not familiar with Windows installers but looks like it should be possible to build one that fetches the latest exe from the website. This would avoid re-making an installer _and_ portable exe every release (the portable exe is required for the builtin update mechanism).

**Comment by sheeepdev** [comment]
Yeah, I only want this because portable programs do have some flaws like I cant set kanmail as my preferred mail client because 1. windows doesn't support portable programs as default, and 2. it probably doesn't even recognize kanmail as a mail client. it would be nice if you could make the installer.

**Comment by sheeepdev** [comment]
Is this still planned?

**Comment by Fizzadar** [comment]
Yes, still want to do this but just haven’t had any time recently to dedicate to Kanmail beyond bug fixes - been a busy few months!

I’m hoping to get to this at the same time as implementing code signing for Windows as well. Now have a working Windows dev setup so just need some time to figure out how to do this, have never made an installer before!

**Comment by Fizzadar** [comment]
Finally making progress on this! Going to use Inno Setup to do it which also supports downloading meaning should only need one installer that always gets the latest version (https://jrsoftware.org/isfaq.php#download).

This is now next up on my radar, hopefully this coming week.

**Comment by Fizzadar** [comment]
v2 has this.
